packages/compiler/src/server/compile-service.ts

```diff
@@ -329,6 +329,15 @@ export function createCompileService({
       return path;
     }
 
+    const entrypoints = clientConfigsProvider?.config?.compile?.entrypoint;
```

Comment:
you also need to go to parent folders to check whether the entrypoints file exists like waht the while(...) do below.